src/ke.cpp

```diff
@@ -647,10 +654,10 @@ class _usersim_emulated_dpc
 
                         l.unlock();
                         _usersim_dispatch_locks[i].lock();
-                        _usersim_current_irql = DISPATCH_LEVEL;
+                        //_usersim_current_irql = DISPATCH_LEVEL;

```

Comment:
commented out code - can you remove if unnecessary, else add an explanatory comment?